### ISSUE TITLE
Bug: getImageURI in Column Chart with long labels - wrongly placed

### ISSUE THREAD

**Issue by orwant** [google/google-visualization-issues#1495]
```
What steps will reproduce the problem? Please provide a link to a
demonstration page if at all possible, or attach code.
1. Placement of the labels in the PNG is incorrect
2.
3.


What component is this issue related to (PieChart, LineChart, DataTable,
Query, etc)?
ColumnChart

Are you using the test environment (version 1.1)?
(If you are not sure, answer NO)
No

What operating system and browser are you using?
Linux - Firefox 26.0 and Chrome 33.0.1750.117


*********************************************************
For developers viewing this issue: please click the 'star' icon to be
notified of future changes, and to let us know how many of you are
interested in seeing it resolved.
*********************************************************

```

Original issue reported on code.google.com by `qofdatabase` on 2014-02-22 20:40:39

<hr>
- _Attachment: chart.png<br>![chart.png](https://storage.googleapis.com/google-code-attachments/google-visualization-api-issues/issue-1495/comment-0/chart.png)_
- _Attachment: [chart.txt](https://storage.googleapis.com/google-code-attachments/google-visualization-api-issues/issue-1495/comment-0/chart.txt)_

**Comment by orwant** [comment]
```
(No text was entered with this change)
```

Original issue reported on code.google.com by `dlaliberte@google.com` on 2014-02-28 17:35:50

**Comment by orwant** [comment]
```
This should be fixed in the 1.1 release. Please take a look.
```

Original issue reported on code.google.com by `jfaller@google.com` on 2014-03-05 15:09:59

**Comment by orwant** [comment]
```
Thanks. Looks fixed to me on the 1.1 release. That is brilliant.
```

Original issue reported on code.google.com by `qofdatabase` on 2014-03-05 15:30:53

<hr>
- _Attachment: fixed.png<br>![fixed.png](https://storage.googleapis.com/google-code-attachments/google-visualization-api-issues/issue-1495/comment-3/fixed.png)_

**Comment by orwant** [comment]
```
Alright. I'm going to call this fixed. It should make the v38 release.

Please note, we try not to break functionality in 1.1, but it might break before 1.1
becomes 1.0.
```

Original issue reported on code.google.com by `jfaller@google.com` on 2014-03-05 19:33:09
